google/cloud/forseti/scanner/scanners/firewall_rules_scanner.py

```diff
@@ -118,15 +117,15 @@ def _output_results(self, all_violations, resource_counts):
                 LOGGER.info('CSV filename: %s', output_csv_name)
 
                 # Scanner timestamp for output file and email.
-                now_utc = datetime.utcnow()
+                now_utc = date_time.get_utc_now_datetime()
 
                 output_path = self.scanner_configs.get('output_path')
                 if not output_path.startswith('gs://'):
                     if not os.path.exists(
                             self.scanner_configs.get('output_path')):
                         os.makedirs(output_path)
                     output_path = os.path.abspath(output_path)
-                self._upload_csv(output_path, now_utc, output_csv_name)
+                base_scanner.upload_csv(output_path, now_utc, output_csv_name)
```

Comment:
That is unless the method doesn't require use of `self` which in this case it doesn't which pylint correctly points out that it shouldn't be included and factored out. Honestly uploading a file shouldn't be in the scanner logic and should be a common utility anyways.

Reply:
See comment above about `@staticmethod`.

Fixing the uploading of the file is outside the scope of this PR.  IOW, we should try to make our PR as self-contained as possible.  This will be removed when Muharem consolidate this functionality into the notifier.

Reply:
done